blueocean-core-js/src/js/fetch.js

```diff
@@ -185,11 +188,13 @@ export const FetchFunctions = {
                     future = future.then(FetchFunctions.checkRefreshHeader);
                 }
 
-                future = future.then(FetchFunctions.checkStatus)
-                    .then(FetchFunctions.parseJSON, FetchFunctions.parseErrorJson);
+                future = future.then(FetchFunctions.checkStatus).then(FetchFunctions.parseJSON, FetchFunctions.parseErrorJson);
```

Comment:
Interesting, I expected each `then` to be on its own line. Is this because it's not exceeding the max line length?

Reply:
maybe because it doesn't have a nested function like here https://github.com/jenkinsci/blueocean-plugin/pull/1474/files#diff-4ef4ba5e27a29ca99e7cf854e77f5805R19